components/PostCard.tsx

```diff
@@ -28,22 +58,36 @@ const toBase64 = (str: string) => Buffer.from(str).toString("base64");
 const PostCard = ({ post }: Props) => {
   const dispatch = useAppDispatch();
 
+  const classNameOfPost =
+    isPostADraft(post) === true || isPostInTheFuture(post) === false
+      ? "posts-list-block posts-list-block-draft-or-future"
+      : "posts-list-block";
+
   return (
     <li>
-      <div className="posts-list-block">
+      {isPostADraft(post) ? <DraftPostMark /> : null}
+      {isPostInTheFuture(post) === false ? <FuturePostMark /> : null}
```

Comment:
It looks like my comment, is lost, but why not && ?

Reply:
bcs post can be in the future but no draft. Or don't u mean that? 
```suggestion
      {isPostADraft(post) && isPostInTheFuture(post) === false ? <DraftPostMark /> <FuturePostMark /> : null}
```

Reply:
Not that, it just a comment got lost

I was suggesting the following change:

```ts
{isPostADraft(post) && <DraftPostMark />}
{isPostInTheFuture(post) === false && <FuturePostMark />}
```

So we don't have write a ternary case for null value

If there are any problems with this method, it's cool to skip it

Reply:
@ntorbinskiy